src/user/user.service.ts

```diff
@@ -151,21 +151,17 @@ export class UserService {
     { name, email, contactPermission, serviceEmailsPermission, signUpLanguage }: CreateUserDto,
     firebaseUid: string,
   ) {
-    try {
-      const createUserObject = this.userRepository.create({
-        name,
-        email,
-        firebaseUid,
-        contactPermission,
-        serviceEmailsPermission,
-        signUpLanguage,
-      });
-      const createUserResponse = await this.userRepository.save(createUserObject);
+    const createUserObject = this.userRepository.create({
```

Comment:
what was the reason for removing the try catches

Reply:
if the catch block only contains `throw error;` then the try/catch is redundant, was a learning for me! These were flagged by eslint once eslint config was fixed in [e68a25d](https://github.com/chaynHQ/bloom-backend/pull/405/commits/e68a25d3581d40f4e92057b5a4321c5a20353969)